### PR TITLE
Added data loading module and test file

### PR DESCRIPTION
**Pull request by acionca**
- Added module to get filename and load data from individual subject-run pairs (`sub-01; run02`) or lists (`[sub-01, sub-02]; run01)`.
- Added test file for the data loading module.

### CODE REVIEW

File: findoutlie/data_load.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    ----------
+    sub_id : int
+        Subject ID to load data from.
+    run_num : int
+        Run number to load.
+
+    Returns
+    -------
+    func_img
+        Nifti image of the functional run.
```
Comment (acionca) [inline findoutlie/data_load.py]:
```suggestion
    sub_id : int or list
        Subject ID(s) to load data from.
    run_num : int or list
        Run number(s) to load.

    Returns
    -------
    func_img
        Nifti image(s) of the functional run(s).
```

File: findoutlie/tests/test_data_load.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    img_list = load_sub_run([1, 2], 2)
+    assert len(img_list) == 2
+    assert type(img_list) == list
+    assert type(img_list[0]) == nifti1.Nifti1Image
+
+    img_list = load_sub_run([1, 2], [1, 2])
+    assert len(img_list) == 4
+    assert type(img_list) == list
+    assert type(img_list[2]) == nifti1.Nifti1Image
+
```
Comment (acionca) [inline findoutlie/tests/test_data_load.py]:
```suggestion    
```

File: findoutlie/tests/test_data_load.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    assert type(img_list) == list
+    assert type(img_list[2]) == nifti1.Nifti1Image
+    
+    
+
+
+if __name__ == "__main__":
+    # File being executed as a script
+    test_get_fname()
+    test_load()
```
Comment (acionca) [inline findoutlie/tests/test_data_load.py]:
```suggestion
```

**Comment by hugofluhr** [comment]
@Cionkito did you see that the tests are failing?

**Comment by acionca** [comment]
Yup, I suggested changes to update the test files. Should be good now

File: findoutlie/tests/test_data_load.py (hunk 2 above)
Comment (hugofluhr) [reply findoutlie/tests/test_data_load.py]: Am I understanding correctly that this test is removed because of the data not being in the repo?

Comment (acionca) [reply findoutlie/tests/test_data_load.py]: Exactly, yes. Maybe we can consider uploading a sample data in the `test `directory to still run those tests (as they did for the outlier detection) or use the existing data.

That probably is for later though.

**Comment by hugofluhr** [comment]
in the future when you need to make changes to a PR you authored, you can apply modifications by adding a commit instead of making suggestions on here.

**Pull request merged**